src/Microsoft.PowerShell.PSReadLine/ConsoleKeyChordConverter.cs

```diff
@@ -323,8 +325,19 @@ internal static char GetCharFromConsoleKey(ConsoleKey key, ConsoleModifiers modi
 
             // get corresponding character  - may be 0, 1 or 2 in length (diacritics)
             var chars = new char[2];
-            int charCount = NativeMethods.ToUnicode(
-                virtualKey, scanCode, state, chars, chars.Length, NativeMethods.MENU_IS_INACTIVE);
+            int charCount = 1;
+            if (_toUnicodeApiAvailable)
```

Comment:
There's a comment in the code that GetCharFromConsoleKey() is needed to support chords like Ctrl+[.  Seems like the best approach here is to defer to the Unix code if this API isn't available.  I'll make that change.